docs/man/git-lfs-ext.1.ronn

```diff
@@ -0,0 +1,25 @@
+git-lfs-ext(1) - View extension details
+==============================================================
+
+## SYNOPSIS
+
+`git lfs ext list` [<name>...]
+
+## DESCRIPTION
+
+Git LFS extensions enable the manipulation of files streams
+during smudge and clean.
+
+## EXAMPLES
+
+* List details for all extensions
+
+    `git lfs ext list`
+
+* List details for the specified extensions
+
+    `git lfs ext list 'foo' 'bar'`
+
```

Comment:
Do we need the `list` command? What about:

```
# list all extensions
git lfs ext

# show the foo and bar extensions
git lfs ext "foo" "bar"
```

Now, maybe we will want sub commands like "add". In that case, how about:

```
# list all extensions
git lfs ext

# show extensions
git lfs ext show 'foo' 'bar'

# add extension (in future PR)
git lfs ext add "foo" ...
```